docs/ML-Agents-Overview.md

```diff
@@ -319,11 +319,11 @@ imitation learning algorithm will then use these pairs of observations and
 actions from the human player to learn a policy. [Video
 Link](https://youtu.be/kpb8ZkMBFYs).
 
-ML-Agents provides ways to both learn directly from demonstrations as well as
-use demonstrations to help speed up reward-based training, and two algorithms to do
-so (Generative Adversarial Imitation Learning and Behavioral Cloning). The
-[Training with Imitation Learning](Training-Imitation-Learning.md) tutorial
-covers these features in more depth.
+The toolkit provides a way for agents to learn directly from demonstrations using
```

Comment:
GAIL can be used for both learning directly from demonstrations AND speeding up RL training. BC can technically do both as well (though we call the BC to speed up "PreTraining").